contracts/acl/ACL.sol

```diff
@@ -145,6 +149,21 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermission(_entity, _app, _role, NO_PERMISSION);
     }
 
+    /**
+    * @dev Revokes all permissions for a specified role if allowed. This requires `msg.sender` to be the the permission manager
+    * @notice Revoke from all the ability to perform actions requiring `_role` on `_app`
+    * @param _app Address of the app in which the role will be revoked
+    * @param _role Identifier for the group of actions in app being revoked
+    */
+    function revokeAll(address _app, bytes32 _role)
+        external
+        onlyPermissionManager(_app, _role)
+    {
+        uint256 newRoleEra = roleEras[roleHash(_app, _role)] + 1;
+        require(newRoleEra >= roleEras[roleHash(_app, _role)], ERROR_ROLE_ERA_INCREMENT);
+        roleEras[roleHash(_app, _role)] = newRoleEra;
```

Comment:
I wonder if using a pointer to `roleEras[roleHash(_app, _role)]` could save some gas here.

Reply:
Do you mean using a storage pointer? I don't think I can on a uint256 value. Maybe I'm missing something though.

Reply:
Oh, yes, you are right, sorry for the confusion. We could still save 2 calls to `roleHash` function, but as it's going to be one less if you remove the `require`, I don't know if that's a big gain.

Reply:
Probably still a decent net gain, since there also wouldn't be another `SLOAD`.

Reply:
Good :) Pushed the optimization.